### ISSUE TITLE
jenkins: Investigate whether javaposse.jobdsl.plugin.ExecuteDslScripts needs to be retained

### ISSUE THREAD

**Issue by sxa** [adoptium/infrastructure#3500]
https://github.com/adoptium/infrastructure/issues/3491 the ExecuteDslScripts directory on the jenkins server needs to be kept. It takes just over 1MB of space and it generating one directory every few minutes.

The ones I've looked at seem to be related to job generation so I'm going to hand this over to @andrew-m-leonard and @smlambert  - are you aware that these logs are being created and would you expect there to be generation activity this frequently? It would be good to understand 
[javaposse.jobdsl.plugin.ExecuteDslScripts.xml.gz](https://github.com/adoptium/infrastructure/files/14873645/javaposse.jobdsl.plugin.ExecuteDslScripts.xml.gz)

**Comment by andrew-m-leonard** [comment]
Yes this is required for Jenkins job generation, our generation jobs use the https://plugins.jenkins.io/job-dsl/ Jenkins plugin, https://github.com/jenkinsci/job-dsl-plugin.
Eg. here:
https://github.com/adoptium/ci-jenkins-pipelines/blob/3101bb087e49336a689ea2d2a352a2c773b212c6/pipelines/jobs/weekly_release_pipeline_job_template.groovy#L7

**Comment by sxa** [comment]
This seems to be down to about 3 per hour now in the general case (outside when pipelines are running) although there was a discussion recently relating to whether a full regen is strictly required on each build.
